studio/components/layouts/AppLayout/EnableBranchingButton/GithubRepositorySelection.tsx

```diff
@@ -52,6 +52,12 @@ const GithubRepositorySelection = ({
   const [repoOwner, repoName] = githubConnection?.metadata.name.split('/') ?? []
 
   const sidePanels = useSidePanelsStateSnapshot()
+  const githubIntegrationAppUrl =
+    process.env.NEXT_PUBLIC_ENVIRONMENT === 'prod'
```

Comment:
There's an actual Vercel env var for 'prod' and 'preview' if anyone can remember it. Better than relying on a custom one.

Reply:
i'll probably keep this first and refactor in another PR cause theres multiple places using NEXT_PUBLIC_ENVIRONMENT

ref for myself: https://vercel.com/docs/projects/environment-variables/system-environment-variables